tier4_system_msgs/msg/DiagNodeStruct.msg

```diff
@@ -0,0 +1,2 @@
+string path
+string name
```

Comment:
Can you give me an example of how to use this?

Reply:
`path` is an arbitrary node name on the graph and `name` is the name used in `/diagnostics`. See PR description for details.